src/test/java/uk/gov/justice/hmpps/casenotes/integration/wiremock/OauthMockServer.java

```diff
@@ -16,12 +17,12 @@ public OauthMockServer() {
 
     public void subGetUserDetails(final AuthToken username) {
         stubFor(
-                WireMock.get(WireMock.urlPathMatching(API_PREFIX + "/user/"+username.name()))
-                        .willReturn(WireMock.aResponse()
+                get(urlPathMatching(String.format("%s/user/%s", API_PREFIX, username.name())))
+                        .willReturn(aResponse()
                                 .withHeader("Content-Type", "application/json")
                                 .withBody("{\n" +
```

Comment:
Would it be cleaner to use map then just serialize to json?

Reply:
this is a test server for mocking responses, so should match what the other application actually responds